consoles/localXvnc.pm

```diff
@@ -31,16 +31,21 @@ sub callxterm ($self, $command, $window_name) {
     my $display = $self->{DISPLAY};
     $command = "TERM=xterm $command";
     my $xterm_vt_cmd = which "xterm-console";
+    my $pid;
     die "Missing 'xterm-console'" unless $xterm_vt_cmd;
     die('Missing "Xvnc"') unless which('Xvnc');
     die('Missing "icewm"') unless which('icewm');
     die('Missing "xterm"') unless which('xterm');
     if ($self->{args}->{log}) {
         mkpath 'ulogs';
-        $command = "script -f ulogs/hardware-console-log.txt -c \"$command\"";
+        $command = "script -af ulogs/hardware-console-log.txt -c \"$command\"";
     }
-    eval { system("DISPLAY=$display $xterm_vt_cmd -title $window_name -e bash -c '$command' & echo \"xterm PID is \$!\""); };
-    die "cant' start xterm on $display (err: $! retval: $?)" if $@;
+    $pid = fork();
+    die unless defined $pid;
```

Comment:
This should die with a proper error message (or the error handling can be avoided if we're using autodie here).

Reply:
There's `use autodie ':all';` in the `localXvnc` module. I've added `die unless defined $pid;` just for consistency with `localXvnc::activate()` which does the same check after `fork()`.

Reply:
I guess we should just remove the `die` line then.

Reply:
Fixed.